woocommerce/wc-admin
# Migrate tasks to task API

Fixes #7661 

* Migrates the tasks to their own classes
* Uses the tasks in the REST API response
* Reduces `Init` class to mostly initialization methods (another follow-up will be needed to migrate admin scripts that don't belong there)

### Detailed test instructions:

1. Enable the `task` feature flag
2. Send a `GET` request to `wp-json/wc-admin/onboarding/tasks`
3. Check that the response is as expected and includes each task and appropriate properties
4. Smoke test the task list
5. Make sure tasks appear as expected
